resources/js/utils/turbolinks.ts

```diff
@@ -25,20 +25,47 @@ export function currentUrlRelative() {
 
 function keepScrollOnLoad() {
   const { pageXOffset, pageYOffset } = window;
-  $(document).one('turbolinks:load', () => window.scrollTo(pageXOffset, pageYOffset));
+  $(document).one('turbo:load', () => window.scrollTo(pageXOffset, pageYOffset));
 }
 
-export function navigate(url: string, keepScroll = false, { action }: TurbolinksAction = { action: 'advance' }) {
+export function navigate(url: string, keepScroll = false, options?: VisitOptions) {
   if (keepScroll) {
     keepScrollOnLoad();
   }
 
-  Turbolinks.visit(url, { action });
+  Turbo.visit(url, options);
 }
 
 export function reloadPage(keepScroll = true) {
   $(document).off('.ujsHideLoadingOverlay');
-  Turbolinks.clearCache();
+  Turbo.cache.clear();
 
   navigate(currentUrl().href, keepScroll, { action: 'replace' });
 }
+
+export function updateHistory(url: string, action: 'advance' | 'replace') {
```

Comment:
just make it `'push' | 'replace'` to match the methods? (and maybe add a type for it)